frontend/src/routes/(authenticated)/project/[project_code]/AddProjectMember.svelte

```diff
@@ -54,11 +54,11 @@
 <FormModal bind:this={formModal} {schema} let:errors>
   <span slot="title">{$t('project_page.add_user.modal_title')}</span>
   <Input
-    id="email"
-    type="email"
-    label={$t('admin_dashboard.column_email')}
-    bind:value={$form.email}
-    error={errors.email}
+    id="usernameOrEmail"
+    type="text"
+    label={$t('admin_dashboard.column_login')}
```

Comment:
Done in commit 64fd5789. I chose to reference the label text from the login page rather than copy it, to ensure that if we change the label on the login page it would be changed here as well. If you think it would be better if those two strings could get out-of-sync, then I can do that instead.